enterprise/coderd/coderdenttest/coderdenttest.go

```diff
@@ -199,6 +199,13 @@ func (opts *LicenseOptions) Valid(now time.Time) *LicenseOptions {
 	return opts
 }
 
+func (opts *LicenseOptions) FutureTerm(now time.Time) *LicenseOptions {
+	opts.NotBefore = now.Add(time.Hour * 24)
+	opts.ExpiresAt = now.Add(time.Hour * 24 * 60)
+	opts.GraceAt = now.Add(time.Hour * 24 * 53)
```

Comment:
Would it make sense to use `AddDate()` here?

Reply:
`AddDate()` is tied to timezones and could be ambiguous around daylight savings switches. Adding hours is not ambiguous.